xkcd_dl/cli.py

```diff
@@ -236,23 +228,22 @@ def set_custom_path(custom_path):
         print("The path does not exist.")
     return path_was_set
 
+
 def show_xkcd(num):
     download_one(read_dict(), num)
-    path = '{current_directory}/xkcd_archive/{name}/'.format(
-        current_directory=WORKING_DIRECTORY, 
-        name=num
-    )
+    path = '{current_directory}/xkcd_archive/{name}/'.format(current_directory=WORKING_DIRECTORY, name=num)
```

Comment:
This looks like so because of the side-by-side view. Pls look at line 234 [here](https://github.com/tusharmakkar08/xkcd-dl/blob/828c2f7e518adecd67dc91f2341c22d598c457d9/xkcd_dl/cli.py).. Can revert this change, if you still feel so. (following 120 char limit)